src/az-purview/deploy/terraform/modules/keyvault/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "resource_group_name" {
+  description = "Resource Group name to host keyvault"
+  type        = string
+}
+
+variable "location" {
+  description = "key vault location"
+  type        = string
+}
+
+variable "keyvault_sku_name" {
+  description = "keyvault sku - potential values Standard and Premium"
```

Comment:
Should there be a validation rule to enforce the allowed values?

Reply:
Good catch